picturae_importer.py

```diff
@@ -41,10 +50,23 @@ def __init__(self, paths, date_string=None):
 
         self.batch_md5 = self.sql_csv_tools.generate_token(starting_time_stamp, self.file_path)
 
-
-
         self.run_all_methods()
 
+    # def setup_logger(self):
```

Comment:
I actually will remove this left over function in the next push, remnant from using a logger handler.